packages/block-editor/src/components/text-transform-control/stories/index.story.js

```diff
@@ -1,33 +1,74 @@
+/**
+ * Internal dependencies
+ */
+import TextTransformControl from '../';
+
 /**
  * WordPress dependencies
  */
 import { useState } from '@wordpress/element';
 
 /**
- * Internal dependencies
+ * Meta configuration for Storybook
  */
-import TextTransformControl from '../';
-
-export default {
+const meta = {
 	title: 'BlockEditor/TextTransformControl',
 	component: TextTransformControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'Control to facilitate text transformation selections.',
+			},
+		},
+	},
 	argTypes: {
-		onChange: { action: 'onChange' },
+		onChange: {
+			action: 'onChange',
+			control: {
+				type: null,
+			},
+			description: 'Callback function when text transform changes',
+			table: {
+				type: {
+					summary: 'function',
+				},
+			},
+		},
+		className: {
+			control: { type: 'text' },
+			description: 'Additional CSS class name to apply',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		value: {
+			control: { type: null },
+			description: 'Currently selected writing mode.',
+			table: { type: { summary: 'string' } },
+		},
 	},
 };
 
-const Template = ( { onChange, ...args } ) => {
-	const [ value, setValue ] = useState();
-	return (
-		<TextTransformControl
-			{ ...args }
-			onChange={ ( ...changeArgs ) => {
-				onChange( ...changeArgs );
-				setValue( ...changeArgs );
-			} }
-			value={ value }
-		/>
-	);
-};
+export default meta;
 
-export const Default = Template.bind( {} );
+/**
+ * Default Story Export
+ */
+export const Default = {
```

Comment:
```suggestion
export const Default = {
```

I think this comment is redundant and can be deleted.